prisma/migrations/20221004074643_add_createdat_and_updatedAt_to_cohort/migration.sql

```diff
@@ -0,0 +1,3 @@
+-- AlterTable
+ALTER TABLE "Cohort" ADD COLUMN     "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+ADD COLUMN     "updatedAt" TIMESTAMP(3);
```

Comment:
This migration will crash the production database. Please delete this file, add a default to the `updatedAt` column on the Cohort model of `@default(now())` and generate a new migration

Reply:
Updated